crates/packager/src/package/appimage/mod.rs

```diff
@@ -88,6 +88,8 @@ pub(crate) fn package(ctx: &Context) -> crate::Result<Vec<PathBuf>> {
     // generate deb_folder structure
     tracing::debug!("Generating data");
     let icons = super::deb::generate_data(config, &appimage_deb_data_dir)?;
+    tracing::debug!("Copying files specified in `deb.files`");
+    super::deb::copy_custom_files(config, &appimage_deb_data_dir)?;
```

Comment:
`deb.files` is something specific to `deb`, don't know if we should use it on `appimage` tbh, why was this needed anyways?

Reply:
Electron needs to inject a bunch of files in the appimage. Appimage and deb share the same data folder so I think it's ok, but we can add a separate config for it too.

Reply:
Yeah let's add a separate config then, better be explicit about it